src/main/java/racingcar/domain/Participant.java

```diff
@@ -2,29 +2,30 @@
 
 public class Participant {
     private static final int MAX_NAME_LENGTH = 5;
+    public static final String MESSAGE_INPUT_PERSONAL_NAME = "개인 참가자 이름을 입력하세요.";
+    public static final String MESSAGE_MAX_NAME_LENGTH = new StringBuilder("개인 참가자 이름의 길이는 ")
+            .append(MAX_NAME_LENGTH)
+            .append("글자를 초과할 수 없습니다.")
+            .toString();
```

Comment:
이 예외 메세지는 값이 가변적으로 바뀌지 않네요. StringBuilder를 사용할 필요는 없어보여요! 
```suggestion
    public static final String MESSAGE_MAX_NAME_LENGTH = String.format("개인 참가자 이름의 길이는 %d글자를 초과할 수 없습니다.",MAX_NAME_LENGTH);
```